packages/ods-react/src/components/pagination/src/components/pagination-page-size-selector/PaginationPageSizeSelector.tsx

```diff
@@ -0,0 +1,47 @@
+import { usePaginationContext } from '@ark-ui/react/pagination';
+import { type FC } from 'react';
+import { Select, SelectContent, SelectControl, type SelectItem } from '../../../../select/src';
+import { PAGINATION_PER_PAGE_OPTIONS } from '../../constants/pagination-per-page';
+import style from './paginationPageSizeSelector.module.scss';
+
+type RenderTotalItemsLabel = (params: { totalItems: number }) => string | number;
+type HandleValueChange = (detail: { value: string[] }) => void;
+
+interface PaginationPageSizeSelectorProp {
+  onSelectChange?: (value: string) => void;
+  renderTotalItemsLabel?: RenderTotalItemsLabel;
+}
+
+const defaultRenderTotalItemsLabel: RenderTotalItemsLabel = ({ totalItems }) => totalItems;
```

Comment:
TBD:
- move to root where we usually set all prop?
- add full english sentence by default?